TM1py/Services/ElementService.py

```diff
@@ -129,6 +129,23 @@ def get_all_leaf_element_identifiers(self, dimension_name: str, hierarchy_name:
         mdx_elements = f"{{ Tm1FilterByLevel ( {{ Tm1SubsetAll ([{dimension_name}].[{hierarchy_name}]) }} , 0 ) }}"
         return self.get_element_identifiers(dimension_name, hierarchy_name, mdx_elements, **kwargs)
 
+    def get_elements_by_level(self, dimension_name: str, hierarchy_name: str, level: int,
+                              **kwargs) -> List:
```

Comment:
better to use `List[str]` here for the type hint

Reply:
thanks will update that..